docs/tools/CLI/troubleshooting.md

```diff
@@ -0,0 +1,14 @@
+
+## Troubleshooting
+
+### Missing Python dependencies
+
+Mbed CLI attempts to automatically installs missing Python dependencies. This step can fail if Mbed CLI does not have permission to install them. If it fails, Mbed CLI will print out a warning and a suggested command to run with the necessary permissions:
```

Comment:
```suggestion
Mbed CLI attempts to automatically install missing Python dependencies. If Mbed CLI doesn't have install permissions, installation fails the Mbed CLI prints out a warning and a suggested command to run with the necessary permissions:
```

Reply:
How about this, "install permissions" is a little imprecise:
```suggestion
Mbed CLI attempts to automatically install missing Python dependencies. If Mbed CLI doesn't have the necessary permissions, installation fails and Mbed CLI prints out a warning and a suggested command to run with the necessary permissions:
```